language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/EnumMemberNodeContext.java

```diff
@@ -32,13 +34,17 @@
  */
 @JavaSPIService("org.ballerinalang.langserver.commons.completion.spi.BallerinaCompletionProvider")
 public class EnumMemberNodeContext extends AbstractCompletionProvider<EnumMemberNode> {
+
     public EnumMemberNodeContext() {
         super(EnumMemberNode.class);
     }
 
     @Override
     public List<LSCompletionItem> getCompletions(BallerinaCompletionContext ctx, EnumMemberNode node)
             throws LSCompletionException {
-        return Collections.emptyList();
+        List<Symbol> visibleSymbols = ctx.visibleSymbols(ctx.getCursorPosition());
+        List<Symbol> filteredSymbols = visibleSymbols.stream()
+                .filter(symbol -> symbol.kind() == SymbolKind.CONSTANT).collect(Collectors.toList());
```

Comment:
Shouldn't this further be filtered to consider only the string constants? Since that's the only valid type of constants for enum members.

Reply:
You can use the `broaderTypeDescriptor()` method in `ConstantSymbol` for this.